### PR TITLE
Deprecation notice and proposal to archive this repo

### PR DESCRIPTION
**Pull request by dbluhm**
As discussed on the Aries WG call today, we're proposing that we archive this repo. This PR adds a deprecation notice to the README prior to that archiving.

cc @ryjones @TelegramSam @burdettadam @swcurran @TimoGlastra

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,24 @@
...
+Issuance and Verification over DIDComm using the Toolbox because we can do it
+with any number of available Issuer, Verifier, and Holder agents.
+
+There are some components of the Toolbox that remain relevant and valuable
+tools during Agent development, such as live composition of DIDComm messages
+and being able to analyze and inspect DIDComm messages in plaintext. These
+concepts will live on in other tools and projects.
+
+If you have questions about this project or replacements for this project, feel
+free to reach out on Hyperledger's Discord: https://discord.gg/hyperledger
```
Comment (TimoGlastra) [inline README.md:20]: Would it be beneficial to add alternatives here?

I know someone (anonyome?) built an UI on top of ACA-Py and there's definitely more in this category. We have Siera Desktop (https://github.com/animo/siera-desktop) which also replaces some parts of the toolbox.

**Review by TimoGlastra — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (swcurran) [reply README.md:20]: Definitely — please add the alternatives that are available.

**Review by swcurran — CHANGES_REQUESTED** [review]
@dbluhm — suggest that you add to this PR the file `.github/settings.yml` to this PR, with appropriate settings.  Here is an example of such a file:

https://github.com/hyperledger-archives/ursa/blob/main/.github/settings.yml

File: README.md (hunk 1 above)
Comment (TelegramSam) [reply README.md:20]: My only concern is keeping the alternatives list current, which we can't do if it's archived. We could include "at the time of archival, here are options. Check with the community for other alternatives." as a way to address the impending staleness.

**Comment by ryjones** [comment]
@dbluhm @swcurran I've added this to the PR

**Review by swcurran — APPROVED** [review]

**Pull request merged**